app/src/test/java/org/xbmc/kore/provider/mediaprovider/AlbumsTest.java

```diff
@@ -57,11 +58,11 @@ public void queryAlbumTest() {
     }
 
     @Test
-    public void queryAlbumsForArtistTest() {
+    public void queryAlbumsForArtistTest() throws Exception {
         Uri uri = MediaContract.AlbumArtists.buildAlbumsForArtistListUri(hostInfo.getId(),
                                                                          TestValues.Artist.artistId);
 
-        Cursor cursor = shadowContentResolver.query(uri, TestValues.Album.PROJECTION, null, null, null);
+        Cursor cursor = client.query(uri, TestValues.Album.PROJECTION, null, null, null);
```

Comment:
Guess this needs to be done in `GenresTest`, `SongsTest` and `ArtistsTest` also.

Reply:
Yeah, still working on it. I didn't have time left to do it yesterday.

Reply:
All green!